tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/konan/tasks/KonanBuildingTask.kt

```diff
@@ -47,16 +47,8 @@ abstract class KonanBuildingTask: KonanArtifactWithLibrariesTask(), KonanBuildin
     val konanVersion
         @Input get() = project.konanVersion.toString(true, true)
 
-    protected abstract fun buildArgs(): List<String>
-
     @TaskAction
-    open fun run() {
-        destinationDir.mkdirs()
-        if (dumpParameters) {
-            dumpProperties(this)
-        }
-        toolRunner.run(buildArgs())
-    }
+    abstract fun run()
```

Comment:
Isn't this plugin deprecated? Is it required to support two-stage compilation then?

Reply:
> Is it required to support two-stage compilation then?

As far as I understand, only for :backend.native:tests. Is it so?

Reply:
Yes, this change is only for tests